tox.ini

```diff
@@ -179,3 +179,30 @@ install_command = pip install --no-deps {opts} {packages}
 commands =
     conda list
     {env:PYTEST_COMMAND} {posargs}
+
+[testenv:pyinstaller]
+description = check that sunpy can be included in a pyinstaller bundle
+changedir = .pyinstaller
+deps =
+    # The developer version of PyInstaller doesn't work yet because it
```

Comment:
Is this copied out of astropy? Is this still true for us, this PR linked here is pretty old.

Reply:
Yeah, I took reference from AstroPy, I think this wouldn't be a problem for Sunpy though.

Reply:
Ideally if we can test with a released version that would be better.

Reply:
Oh yeah sorry I undid these changes accidentally.